pinot-connectors/pinot-flink-connector/pom.xml

```diff
@@ -51,7 +51,18 @@
     </dependency>
     <dependency>
       <groupId>org.apache.flink</groupId>
-      <artifactId>flink-streaming-java_${scala.compat.version}</artifactId>
+      <artifactId>flink-streaming-java</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.flink</groupId>
+      <artifactId>flink-java</artifactId>
+      <exclusions>
```

Comment:
Ideally we want to do the exclusion within the root pom. Which 2 libraries have conflict? Should we explicitly manage the version of this library?

Reply:
@sullis Can you address this? We can move the exclusion to the root pom?

Reply:
Can you address this comment?

Reply:
I'll work on a fix later today. I'm going outside for a hike 😎

Reply:
> Can you address this comment?

I moved the exclusion to the root pom.